tests/test_command.py

```diff
@@ -34,7 +34,11 @@ def test_command_override(example1_config):
 
 def test_nonexistent_interpolation_keys():
     empty_parameter_map = ParameterMap(parameters={}, values={})
-    interp_command = build_command(['Where are the ${shell_unicorns}? The {parameters} are here!'], empty_parameter_map)
+    interp_command = build_command(
+        ['Where are the ${shell_unicorns}? The {parameters} are here!'],
+        empty_parameter_map,
+        special_interpolations={}
+    )
```

Comment:
Doesn't look like this needed to be changed since `special_interpolations` has a default?

Reply:
Not needed, no, the field is optional mostly because it is theoretically a public API but considering it mandatory internally should be ok.

Reply:
I dropped it off the old test and added a new test specifically for special interpolations.